.github/workflows/cypress.yml

```diff
@@ -9,4 +9,8 @@ jobs:
       - name: Install dependencies
         run: npm install
       - name: Component tests
-        run: npm run test:ct
\ No newline at end of file
+        run: npm run test:ct
+      - name: Jest tests
+        run: npm run test
```

Comment:
Let's make it the same way as was done for vuln4shift-frontend: swap the order of these two steps: so that Jest tests run before component tests.